### ISSUE TITLE
scrollbar issue, if sidebar is on right side

### ISSUE THREAD

**Issue by nean-and-i** [gnunn1/terminix#515]
the scrollbar is not moveable via mouse, if sidebar is on the right side.
I'm able to scroll eg. via scroll wheel,  but I'm not able to grab scrollbar with mouse.
If sidebar is on left side (default) issue is gone.

terminix 1.3.0 (but packagename is still 1.2?)

installed from:
ppa:terminix/daily

that requires for some reason:
ppa:webupd8team/terminix

**Comment by gnunn1** [comment]
Which Ubuntu version are you using 16.04 or 16.10? I had to adjust the scrollbar CSS in 16.10 to make it look right and it's possible I broke this.

**Comment by nean-and-i** [comment]
I'm using Ubuntu Gnome 16.04 LTS.

**Comment by gnunn1** [comment]
I can reproduce, interestingly once I open and close the sidebar at least once it works fine for me thereafter.

**Comment by gnunn1** [comment]
This works fine in 16.10, O'm thinking this is a bug in Ubuntu and their theme, as a result closing and marking as won't fix.

**Comment by nean-and-i** [comment]
Hi,
I've tested this with various gtk themes on Ubuntu Gnome 16.04 LTS such as like ARC, Numix and default Adwaita.
The issue is present on all themes.

also I get following message in stdout whe trying to move the scrollbar:

2016-10-13T15:56:15.689:sidebar.d:onButtonPress:83 ** Sidebar button press
2016-10-13T15:56:16.903:sidebar.d:onButtonPress:83 ** Sidebar button press


Can you debug and prove this is a bug in Ubuntu?
If so, please provide detailed information, so I can raise a case at Ubuntu/Launchpad.

Thank you.

**Comment by gnunn1** [comment]
Actually I can't reproduce this anymore on the Master branch, it works as expected now on 16.04.

I see you were compiling terminix, can you try building from master and see if you still have this problem.

**Comment by nean-and-i** [comment]
Alright, I've downloaded terminix from https://github.com/gnunn1/terminix/releases/tag/1.3.0 and replaced the binary from the current PPA installation with your version from the zip.

so, I'm using now your original binary and can confirm that it is working as expected, no scrollbar issue anymore!

It seems that the PPA maintainer of ppa:terminix/daily and ppa:webupd8team/terminix use a different method for building packages that leads to this issue.

I also checked the version from copr repository heikoada-terminix on Fedora 24, and this one works fine as well.

Sorry for bothering the wrong people, your build seems to be fine.

I  hope that the maintainer and PPA issue will be solved soon and it become stable and reliable for future builds of terminix.

I really felt in love with your terminix, I heavily use it for my daily work, it is a great replacement of terminator, and very close to iTerm2.

thanks,
much appreciate
